assets/css/main.css

```diff
@@ -246,46 +246,93 @@ html, body {
     list-style: none;
 }
 
-/* index thumbnails */
-.course-thumbnails .advert-thumbnail,
-.course-thumbnails .course-thumbnail {
-    height: 330px;
+/* index frontpage */
+  
+.frontpage-panel {
+    margin: 3%;
+    margin-top: 30px;
+}
+
+.frontpage-panel .panel-heading {
+    border-bottom: solid 1px #eee;
+    font-weight: 500;
+}
+
+.frontpage .card {
+    margin-bottom: 20px;
+    border: solid 1px #ccc;
     position: relative;
+    height: 355px;
+    border-radius: 4px;
+    padding: 4px;
 }
-.course-thumbnails .advert-thumbnail {
+
+.frontpage .advert-card {
     background-color: #d9edf7;
     border-color: #bce8f1;
 }
-.course-thumbnails .advert-thumbnail .image,
-.course-thumbnails .course-thumbnail .image {
-    height: 130px;
-    overflow: hidden;
-    text-align: center;
+
+.frontpage a:hover {
+    text-decoration: none;
 }
-.course-thumbnails .course-thumbnail .image {
-    background: #eee;
+
+.frontpage .card-clickable:hover {
+    filter: brightness(85%);
+}
+
+.frontpage .advert-card:hover h3, p,
+.frontpage .card-clickable:hover h3, p {
+    color: #333;
 }
-.course-thumbnails .advert-thumbnail .image img,
-.course-thumbnails .course-thumbnail .image img {
+
+.frontpage .card-img-top {
+    object-fit: cover;
     width: 100%;
-    background: #eee;
+    height: 145px;
+    background-color:#eeeeee;
+}
+
+.frontpage .course-image {
+    color: #fff;
 }
-.course-thumbnails .advert-thumbnail .image .glyphicon,
-.course-thumbnails .course-thumbnail .image .glyphicon {
+
+.frontpage .course-image i,
+.frontpage .advert-image i {
     font-size: 90px;
+    position: absolute;
+    left: 34%;
+    top: 7%;
```

Comment:
I'm not sure why, but when I played around with the hover, I noticed that the closest anchor for `position: absolute` changes. The following patch made it behave a bit better:

```diff
diff --git a/assets/css/main.css b/assets/css/main.css
index ddc587b..bda1ee4 100644
--- a/assets/css/main.css
+++ b/assets/css/main.css
@@ -293,6 +293,7 @@ html, body {
 }
 
 .frontpage .course-image {
+    position: relative;
     color: #fff;
 }
 
@@ -301,7 +302,7 @@ html, body {
     font-size: 90px;
     position: absolute;
     left: 34%;
-    top: 7%;
+    top: 15%;
 }
 
 .frontpage .card-body {
```

Reply:
Oh, that's strange. I did not notice that, I will make the patch

Reply:
This patch has been made now